### ISSUE TITLE
[Bug] Generated password is not copied to clipboard on click

### ISSUE THREAD

**Issue by Te-Z** [linagora/twake-drive-legacy#609]
**Describe the bug**

After creating a password, when I click the box with the password it is not copied to clipboard

Tested on Chrome https://drive.stg.lin-saas.com/

**To Reproduce**
1. I click “…” next to a folder or a file on my drive
2. I select Share
3. I select access rights (read, write)
4. Once I set access rights to anything other than “No access”, advanced settings for the link and the “Copy” button are displayed
5. I check the box “Password”
6. An auto-generated password is displayed there
7. I can change the password
8. When I click the box with the password, is not copied to clipboard

**Expected behavior**

When I click the box with the password, it is copied to clipboard. I see a toast message

**Screenshots**

[Capture vidéo du 22-07-2024 12:47:20.webm](https://github.com/user-attachments/assets/be6040a7-b8f8-4310-8c91-a74da77c73ca)

**Comment by ericlinagora** [comment]
I don't think we should be overwriting the user's clipboard, if the user wants to copy the password, the user can do it in the normal ways stuff is copied. People keep valuable information in their clipboard, replacing it without their request is risky.

**Comment by Popokok** [comment]
How about adding a button to give user an option to copy to clipboard?

**Comment by ericlinagora** [comment]
Sounds good to me
